src/Andre/SoulsFormats/SoulsFormats/Formats/MSB/MSBE/PartsParam.cs

```diff
@@ -2179,7 +2192,7 @@ private protected override void ReadTypeData(BinaryReaderEx br)
                     UnkT40 = br.ReadSingle();
                     br.AssertInt32(0);
                     EnableFastTravelEventFlagID = br.ReadUInt32();
-                    UnkT4C = br.AssertInt16([0, 1]);
+                    UnkT4C = br.AssertInt16([0, 1, 2]);
```

Comment:
This looks like it should be a field (probably enum)
Honestly I don't know why tk ever wrote assert with options in the first place

Reply:
Turns out I wrote this while quite tired, and it is indeed a field